React/my-app/src/components/MultiStateButton/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./styles.css";
+import { WISHLIST, ALQUILAR } from "./strings.js";
+
+const MultiStateButton = ({ available, text, disabled, onClick }) => {
+  return (
+    <div className="button_box">
+      <h1 className="unavailable_text">{text}</h1>
+      <button
+        className={`button ${!available && "wishlist-button"}`}
+        disabled={disabled}
+        onClick={onClick}
+      >
+        {available ? ALQUILAR : WISHLIST}
+      </button>
+    </div>
+  );
+};
+
+export default MultiStateButton;
```

Comment:
I would move this component to `book_detail/components/` and give it a proper name since it's only used there (Like `RentButton`?)

Reply:
Another approach would be to move all the logic to the book detail component and keep the MultiState button generic.